File: Examples/CSharp/AsposePDF/Images/DeleteImages.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;

namespace CSharp.AsposePdf.Images
{
    public class DeleteImages
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_Images();

            //open document
            Document pdfDocument = new Document(dataDir+ "DeleteImages.pdf");

            //delete a particular image
            pdfDocument.Pages[1].Resources.Images.Delete(1);

            //save updated PDF file
            pdfDocument.Save(dataDir + "DeleteImages_out.pdf");
            
            
        }
    }
}

File: Examples/VisualBasic/AsposePDF/Images/DeleteImages.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf
Namespace AsposePDF.Images
    Public Class DeleteImages
        Public Shared Sub Run()
            ' ExStart:DeleteImages
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_Images()

            ' Open document
            Dim pdfDocument As New Document(dataDir & Convert.ToString("DeleteImages.pdf"))

            ' Delete a particular image
            pdfDocument.Pages(1).Resources.Images.Delete(1)

            dataDir = dataDir & Convert.ToString("DeleteImages_out_.pdf")
            ' Save updated PDF file
            pdfDocument.Save(dataDir)
            ' ExEnd:DeleteImages
            Console.WriteLine(Convert.ToString(vbLf & "Images deleted successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace
